sync/class.jetpack-sync-actions.php

```diff
@@ -74,6 +74,10 @@ static function init() {
 		if ( ! wp_next_scheduled ( 'jetpack_send_db_checksum' ) ) {
 			wp_schedule_event( time(), 'hourly', 'jetpack_send_db_checksum' );
 		}
+
+		// Sync Master User Role Changes
+		require_once dirname( __FILE__ ) . '/class.jetpack-sync-users.php';
```

Comment:
Note that this code will never run if the filter `jetpack_sync_sender_should_load` (above) returns false. Is that your intended behaviour?

I imagine so, but I'm not 100% sure.